CMakeLists.txt

```diff
@@ -234,7 +234,7 @@ set(original_skip_samples ${skip_samples})
 set(run_e2e_tests OFF)
 set(run_unittests OFF)
 set(skip_samples ON)
-set(use_cppunittest ON)
+set(use_cppunittest OFF)
```

Comment:
turn this off for all platforms...or only win32?

Reply:
this switch only affects WIN32 in the tests.
https://github.com/Azure/azure-c-shared-utility/blob/341c40b8ce15d2372d9407ef537b0abe45c452d4/configs/azure_c_shared_utilityFunctions.cmake#L232-L240